apps/researcher/src/messages/nl/messages.json

```diff
@@ -24,7 +24,7 @@
   },
   "Tabs": {
     "objects": "Objecten zoeken",
-    "persons": "Personen zoeken"
+    "persons": "Personen en Instellingen zoeken"
```

Comment:
Lowercase `I`, `instellingen`?

Reply:
An afterthought: is the section limited to 'instellingen' (i.e. it's not about 'organisaties' in general)?

Reply:
There has been a discussion on Slack about the Dutch translation. I do not know enough about the scope of this section to decide on the correct translation. So I invite you to ask your questions there. The reasoning from Valentin was: On the NMVW collection website, the term used is 'Personen en instellingen'. This is the term Marjolijn prefers. With the reasoning that "I believe Personen and Instellingen would be good to use. 'Betrokkenen' suggests some level of involvement which might not always be there. For instance if an artwork is inspired by another artists, the latter one probably has nothing to do with this piece. So I would prefer a bit more neutral/broader term like Personen and Instellingen."